learn/appchains/tx-fees.md

```diff
@@ -0,0 +1,116 @@
+
+This section covers the fundamentals of transaction fees for Tanssi Appchains. It first covers the underlying transaction fee architecture and how it is adapted to a fully EIP-1559-compliant model for EVM Appchains.
+
+
+## Baseline Fees Calculation {: #baseline-fees }
+
+Any call that modifies the state of an Appchain has a transaction fee associated with it. This fee is usually associated with the execution cost of the transaction, like _gas_ and _gasPrice_ in EVM-compatible chains. 
+
+Tanssi Appchains use a weight-based mechanism for fees of the [modular framework](/learn/framework/){target=\_blank} also accounts for storage input/outputs. However, Tanssi EVM-compatible Appchains support legacy and [EIP-1559 compatible](https://eips.ethereum.org/EIPS/eip-1559){target=\_blank} transaction pricing mechanisms.
+
+This section outlines all the different concepts associated with transaction fees for Tanssi Appchains.
+
+### Weight {: #baseline-weight}
+
+Broadly speaking, weight refers to the execution time it takes to validate a block, measured in picoseconds. Weight is divided into two separate variables:
+
+- **refTime** - corresponds to the weight associated with computation time and database reads/writes
+- **proofSize** - corresponds to the weigth associated to the size of the _Proof-Of-Validity_ (or _PoV_ for short). The _PoV_ is associated with the relevant state of a transaction, and it is what the Appchain block producer shares with the relay chain validator to get its block finalized as part of the [Appchain transaction flow](/learn/appchains/overview/#appchain-transaction){target=\_blank}
+
+To find the weights for all function calls, they are benchmarked in a system with reference hardware, and the approximate values of _refTime_ and _proofSize_ are set. This process is repeated for all function calls that consume blockspace and affect the _PoV_.
+
+For transactions in which the fees are handled by [Transaction Payment](https://docs.rs/pallet-transaction-payment/latest/pallet_transaction_payment/){target=\_blank} module, all weight-based parameters are then passed through a _WeightToFee_ algorithm that converts all to a final value, deducted from the sender's account when executing the function call. The algorithm can be customized, but Tanssi Appchains have a constant value set.
+
+For EVM transactions, gas is converted to weight through a _GastoWeight_ algorithm so that all EVM calls can be mapped to block execution time. Nevertheless, fees are handled at an EVM execution level.
```

Comment:
same question here about GastoWeight, at the very least I think 'to' should be capitalized to be consistent with WeightToFee